packages/components/pie-cookie-banner/locales/fr.js

```diff
@@ -0,0 +1,40 @@
+export default {
```

Comment:
What's the difference between `fr.js` and `fr-fr.js`?

Reply:
`fr.js` is a global translation for all countries, i.e Canada would fallback to `fr.js` if supplied `fr-ca`, but `fr-fr.js` and `en-fr.js` are bespoke for France only (_slight wording changed for legal purposes_)